docs/sources/k6-studio/components/test-validator.md

```diff
@@ -36,6 +39,12 @@ There are two ways to validate a test:
 
 After you click **Validate script**, k6 Studio runs one iteration of the test script and shows you the results of the test as the script is executed.
 
+## Run a test in Grafana Cloud
+
+After validating a script, you can execute the same test, but using the scenarios you have configured in the **Generator** -> **Load profile** tab, to run a performance test and see how your application behaves in a real-world scenario.
```

Comment:
is `scenario` the correct word ?
I see it mentioned as distribution in the generated script but I'm not 100% sure on how to word it here